FirstProject/new-evaluation.html

```diff
@@ -89,26 +87,35 @@ <h4 style="color:white;">Senior</h4>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">Should the candidate be hired?</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="The type of project that is suitable for the candidate
-Is guidance required for the candidate
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="The type of project that is suitable for the candidate-->
+<!--Is guidance required for the candidate-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">General Impression</h2>
-            <textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px;" placeholder="*required
-" ></textarea>
+            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="*required-->
+<!--" ></textarea>-->
+            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+
+            </div>
         </div>
 
         <div class="row">
             <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & Soft Skills</h2>
```

Comment:
For example, in this particular case, even though the browser would have expected to see the encoded `&amp;` HTML entity instead of just the literal `&` character. Other possibilities include automatic introduction of appropriate closing tags where they're missing (e.g., `<div id="valid"><span id="invalid">This tag isn't closed</div>, but its parent is`), ignoring or defaulting invalid attributes and who knows how many others that your typical web developer might never even think of...